test/ui/Pages/HomePage.ts

```diff
@@ -0,0 +1,29 @@
+import { WebDriver, By, WebElement } from 'selenium-webdriver';
+
+class HomePage {
+    private driver: WebDriver;
+
+    constructor(driver: WebDriver) {
+        this.driver = driver;
+    }
+
+    async mainPage(): Promise<void> {
+        await this.driver.get(process.env.HOMEPAGE_URL_LOCAL as string);
```

Comment:
1. These should be HOMEPAGE_URL
2. Have you added HOMEPAGE_URL as a secret in the GitHub repo which points to the AWS link?

Reply:
done